pkg/secrethub/credentials/readers.go

```diff
@@ -52,3 +55,23 @@ func FromString(raw string) Reader {
 		return []byte(raw), nil
 	})
 }
+
+func fromDefault() Reader {
```

Comment:
`fromDefault` never returns a passphrase but always a credential wrapped in a `Reader`. So should we then change the name to reflect that meaning?

Reply:
Agreed with a rename to `credentialFromDefault()`?